notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/ConnectionString.java

```diff
@@ -88,4 +101,8 @@ public String toString() {
 
         return builder.toString();
     }
+
+    private static boolean isNullOrWhiteSpace(String str) {
+        return str == null || str.trim().equals("");

```

Comment:
I can't believe the Java standard library doesn't have something like this already built in 😆. A quick search didn't yield anything better.